starwars/src/components/Character.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const Character = props => {
+  return (
+    <div className='char-container'>
+    <div className="character">
+      <div className="character-name"> 
+      <ul>
+      <li>{props.character.name}</li>
+      </ul>
```

Comment:
Since you're only rendering one list item, it would be cleaner to simply put `props.character.name` inside a header tag instead of wrapping it in a `ul` and `li`.
Ex.
```suggestion
      <h3>{props.character.name}</h3>
```